connector/src/main/java/org/geysermc/connector/utils/FireworkColor.java

```diff
@@ -44,7 +45,26 @@
     LIGHT_BLUE((byte) 12, 6719955),
     MAGENTA((byte) 13, 12801229),
     ORANGE((byte) 14, 15435844),
-    WHITE((byte) 15, 15790320);
+    WHITE((byte) 15, 15790320),
+
+    // Bukkit colors
+    BUKKIT_WHITE((byte) 15, 0xFFFFFF),
```

Comment:
Can we order these in some way? Perhaps by the order listed in https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Color.html 
Also - may be a good idea to link that as a comment.

Reply:
They are ordered in the same order as they are declared in the `org.bukkit.Color` class, if you scroll down on the page you linked they are in that order. Also will add the link to the page